.github/workflows/ci.yml

```diff
@@ -17,13 +17,14 @@ jobs:
                     - 14
                     - 16
                     - 18
+                    - 20
                 serverless:
                     - "2.36.0"
                     - "2"
                     - "latest"
         steps:
             - uses: actions/checkout@v2
-            - uses: actions/setup-node@v2
+            - uses: actions/setup-node@v3
               with:
                   node-version: ${{ matrix.node }}
             - uses: actions/cache@v2
```

Comment:
@mnapoli maybe here we could use the built-in cache of `actions/setup-node@v3`

Reply:
Indeed that would be great as a separate PR!